tools/ci_setup.ml

```diff
@@ -8,6 +8,90 @@
 
 module StringSet = Set.Make (String)
 
+module In_channel = struct
```

Comment:
You can remove this. The wasm_of_ocaml CI will only work with OCaml >= 4.14, since we need a recent version of ppx_expect, ppx_inline_test and base, which don't work with older versions of OCaml.